app/scripts/metamask-controller.js

```diff
@@ -508,6 +508,7 @@ module.exports = class MetamaskController extends EventEmitter {
       })[0]
     })
     .map(number => number.div(GWEI_BN).toNumber())
```

Comment:
Is this dividing `number`? I do not believe `.div()` is a JS function?

Reply:
The number in this context is a BN.

Reply:
`EthQuery - RPC Error - Cannot read property 'div' of undefined`